pydantic_ai_slim/pydantic_ai/providers/anthropic.py

```diff
@@ -39,7 +42,10 @@ def client(self) -> AsyncAnthropicClient:
         return self._client
 
     def model_profile(self, model_name: str) -> ModelProfile | None:
-        return anthropic_model_profile(model_name)
+        profile = anthropic_model_profile(model_name)
+        if profile:
```

Comment:
I liked the `ModelProfile.from_profile(profile).update(...)` (or `ModelProfile(...).update(profile)`) approach you had before better exactly because it also does the right thing when `anthropic_model_profile` returns `None`